server/concepts/Tournament.ts

```diff
@@ -123,13 +124,64 @@ export default class Tournament implements TournamentWithoutMS {
     /* eslint-enable @typescript-eslint/no-non-null-assertion */
   }
 
-  public static async getById(tournamenId: string, token: string) {
-    const tournament = await TournamentRepository.getById(tournamenId);
+  public static async getById(tournamentId: string, token: string) {
+    const tournament = await TournamentRepository.getById(tournamentId);
     if (!tournament) return null;
     const enrichedTournament = await Tournament.enrichWithMSUsers(tournament, token);
+    // const teams = this.countPointsPerTeam(enrichedTournament);
+    // console.log(teams);
     return enrichedTournament;
   }
 
+  public static countPointsPerTeam(enrichedTournament: ITournament) {
+    // teams: [
+    //   {
+    //     name: "Team 1",
+    //     wins: "1",
+    //     draws: '0',
+    //     loses: '1',
+    //     points: '3',
+    //     matchesPlayed?
+    //   },
+    //   {
+    //     ...
+    //   }
+    // ]
+    const teams: IPointsPerTeam[] = [];
+    enrichedTournament.teams.forEach((team: ITeam) => {
+      let points = 0;
+      let draws = 0;
+      let loses = 0;
+      let wins = 0;
+
+      // eslint-disable-next-line no-restricted-syntax
+      for (const match of enrichedTournament.matches) {
+        if (match.score.final.a === -1 || match.score.final.b === -1) continue;
+        // eslint-disable-next-line no-nested-ternary
+        const teamIdentifierLetter = team.id === match.teamA?.id ? 'a' : (team.id === match.teamB?.id ? 'b' : undefined);
+        if (!teamIdentifierLetter) continue;
+        if (match.score.final[teamIdentifierLetter] > match.score.final[teamIdentifierLetter === 'a' ? 'b' : 'a']) {
+          points += 3;
+          wins++;
+          continue;
+        }
+        if (match.score.final[teamIdentifierLetter] === match.score.final[teamIdentifierLetter === 'a' ? 'b' : 'a']) {
+          points++;
+          draws++;
+          continue;
+        }
+        if (match.score.final[teamIdentifierLetter] < match.score.final[teamIdentifierLetter === 'a' ? 'b' : 'a']) {
+          loses++;
+          continue;
+        }
```

Comment:
```suggestion
        if (teamIdentifierLetter) {
          if (match.score.final[teamIdentifierLetter] > match.score.final[teamIdentifierLetter === 'a' ? 'b' : 'a']) {
            points += 3;
            wins++;
          }
          else if (match.score.final[teamIdentifierLetter] === match.score.final[teamIdentifierLetter === 'a' ? 'b' : 'a']) {
            points++;
            draws++;
          }
          else if (match.score.final[teamIdentifierLetter] < match.score.final[teamIdentifierLetter === 'a' ? 'b' : 'a']) {
            loses++;
          }
        }
```
I think we can avoid `continue`, will it be simpler?

Reply:
It is bad practice to use `else if`'s. `Continue` is much better

Reply:
Any arguments? idk. Why is using an "else if" bad practice? Imo this makes only some changes in visual code side
https://stackoverflow.com/a/33776419

Reply:
You answered yourself with this link :) In this situation there is no particular differenece between continue and else if so I will leave it as it is. In more complicated cases, continue is better.

Reply:
According to what author from your link says :)

Reply:
Yep, but i thought is not complicated loop xD Relative opinion :P